.travis.yml

```diff
@@ -31,6 +31,30 @@ matrix:
         - echo '{"ipv6":true,"fixed-cidr-v6":"2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
         - sudo service docker restart
 
+    - rvm: 2.5.1
+      env:
+        - BEAKER_PUPPET_COLLECTION=puppet5
+        - BEAKER_setfile=centos8-64{hostname=centos8-64.example.com}
+        - LANG=C.UTF-8
```

Comment:
This has no effect and will be dropped in the next modulesync, right? So best to leave it out?

Reply:
Dropped it. I suppose we can leave the annoying warnings for now